SubmittedAssignments/Shirish_Panchal/Preview.html

```diff
@@ -0,0 +1,61 @@
+    <!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&family=Square+Peg&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Freeman&display=swap" rel="stylesheet">
+    <title>Assignemnt_Certificate</title>
+</head>
+<body style="background-color: black;">
+    <!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+        
+        <div class="certificate-background">
+            <div class="hrline"></div>
+            <div class="vrline"><hr></div>
+            <div class="blue_box"></div>
+            <div class="line01"></div>
+            <div class="line1"></div>
+            <div class="line2"></div>
+        </div>
+
+        <div class="certificate-content">
+            
+            <div class="title">C E R T I F I C A T E</div>
+            <div class="subtitle"> OF APPRECIATION</div>
+            <div class="sr-no">Certificate Number: 1234</div>
+            <div class="initial-content">This Certificate is proudly presented to </div>
+            <div class="Candidate-name"> Diana Richard's </div>
```

Comment:
change there **Candidate-name** to  **candidate-name** check (capital **C** to small **c** )